sumy/utils.py

```diff
@@ -49,15 +50,20 @@ def expand_resource_path(path):
 
 
 def get_stop_words(language):
-    path = expand_resource_path("stopwords/%s.txt" % language)
-    if not exists(path):
+    try:
+        stopwords_data = pkgutil.get_data("sumy", "data/stopwords/%s.txt" % language)
```

Comment:
I think you haven't correctly unsderstood my problem.
In our team we use buildout to develop, build and distribute packages.
Buildout use egg format of python packages. 
In case of egg format expand_resource_path returns paths link '/usr/local/lib/python3.5/site-packages/sumy-0.4.1a0-py3.5.egg/sumy/data/stopwords/english.txt' and it's not a legal file (because of .egg is zip-archive).

For this cases pkgutil ( https://docs.python.org/3.5/library/pkgutil.html#pkgutil.get_data )is single known for me standard way to get access to distributed package data.
Of course I can explicitely add possibility to read in-archive pathes for reading data into your code (it you will insist on it) - but I think that this is usage of standard pkgutil is more pretty way to solve problem.